lib/iris/cube.py

```diff
@@ -1679,6 +1836,10 @@ def summary(self, shorten=False, name_padding=35):
             vector_derived_coords = [coord for coord in derived_coords if
                                      id(coord) not in scalar_coord_ids]
 
+            # cell measures
+            vector_cell_measures = [cm for cm in self.cell_measures()
+                                    if cm.shape != (1,)]
```

Comment:
@marqh What if there is a cell measure in the cube with a shape of `(1,)` ... is this not dealt with anywhere ?